bindings/python/build-manylinux-wheel.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -ex
+cd /python
+
+# Compile wheel
+# https://github.com/pypa/manylinux/issues/49
+rm -rf build
+/opt/python/cp37-cp37m/bin/python setup.py bdist_wheel --universal
```

Comment:
cp37-cp37m? I guess they renamed things for 2010? Is there a non-'m' version?

Reply:
These are the available versions:
```
+ ls -la /opt/python/
total 8
drwxr-xr-x 2 root root 4096 Aug 15 06:40 .
drwxr-xr-x 1 root root 4096 Aug 15 06:26 ..
lrwxrwxrwx 1 root root   34 Aug 15 06:28 cp27-cp27m -> /opt/_internal/cpython-2.7.16-ucs2
lrwxrwxrwx 1 root root   34 Aug 15 06:30 cp27-cp27mu -> /opt/_internal/cpython-2.7.16-ucs4
lrwxrwxrwx 1 root root   29 Aug 15 06:33 cp34-cp34m -> /opt/_internal/cpython-3.4.10
lrwxrwxrwx 1 root root   28 Aug 15 06:35 cp35-cp35m -> /opt/_internal/cpython-3.5.7
lrwxrwxrwx 1 root root   28 Aug 15 06:38 cp36-cp36m -> /opt/_internal/cpython-3.6.9
lrwxrwxrwx 1 root root   28 Aug 15 06:40 cp37-cp37m -> /opt/_internal/cpython-3.7.4
```

Note that since our package doesn't have any extensions I don't think it matters which python we use.

Reply:
Agreed. Thanks for clarifying.